app/scripts/modules/core/src/api/network.interceptor.ts

```diff
@@ -72,6 +75,16 @@ export class NetworkInterceptor implements IHttpInterceptor {
         }, (retryCount + 1 + Math.random()) * 1000);
       });
     }
+
+    if (status === 401 && SETTINGS.feature.iapRefresherEnabled) {
+      $http.get('/_gcp_iap/do_session_refresh').then(() => {
```

Comment:
I know this is feature-flagged, but ideally the core Deck library would be unaware of any cloud providers or platforms. 

You could create a separate `IHttpInterceptor` inside either the GCE or App Engine module that does this same check.

Reply:
Done